### PR TITLE
Create Task6.py

### PR DESCRIPTION
**Pull request by ViolettaKurilina**

### CODE REVIEW

File: Tasks/Kurilina/Task_6/Task_favorite_6.py [hunk 1]
```diff
@@ -1,2 +1,17 @@
...
+    
+    final_word = ''
+    if i % 3 == 0:
+        final_word+=word[:4]
+    if i % 5 == 0:
+        final_word+=word[4:]
+    if not final_word:
+        print(i) 
+        continue
+    print(final_word)
```
Comment (EugeneDenkevich) [inline Tasks/Kurilina/Task_6/Task_favorite_6.py:17]: realy smart move i think. =)

File: Tasks/Kurilina/Task_6/Task_favorite_6.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+
+word = "FizzBuzz"
+print(*(word[(0,4,4)[x%3]:4 if x % 5 else 8] or x for x in range(1,101)))
```
Comment (MikitaTsiarentsyeu) [inline Tasks/Kurilina/Task_6/Task_favorite_6.py:4]: it is really interesting

**Review by MikitaTsiarentsyeu — APPROVED** [review]
good job, the score is 100

File: Tasks/Kurilina/Task_6/Task_favorite_6.py (hunk 1 above)
Comment (ViolettaKurilina) [reply Tasks/Kurilina/Task_6/Task_favorite_6.py:17]: Thanks)

File: Tasks/Kurilina/Task_6/Task_favorite_6.py (hunk 2 above)
Comment (ViolettaKurilina) [reply Tasks/Kurilina/Task_6/Task_favorite_6.py:4]: I think so too) but it was found on the internet

**Pull request merged**